packages/dds/matrix/src/matrix.ts

```diff
@@ -591,14 +669,17 @@ export class SharedMatrix<T = any>
 				new ObjectStoragePartition(storage, SnapshotPath.cols),
 				this.serializer,
 			);
-			const [cellData, pendingCliSeqData] = await deserializeBlob(
+			const [cellData, pendingCliSeqData, cellLastWriteTracker] = await deserializeBlob(
 				storage,
 				SnapshotPath.cells,
 				this.serializer,
 			);
 
 			this.cells = SparseArray2D.load(cellData);
 			this.pending = SparseArray2D.load(pendingCliSeqData);
+			if (this.cellLastWriteTracker !== undefined) {
```

Comment:
It's never undefined (based on inspecting code) and we never check for undefined in other places.
Did you mean to check cellLastWriteTracker to be not a string? It will never be undefined, but null (as serializing undefined results in null when using array values)

Reply:
My mistake. I wanted to check for cellLastWriteTracker instead of this. cellLastWriteTracker for older summaries. But I think we are not bothered about that since it will be a separate dds for now.

Reply:
Removed it